### ISSUE TITLE
Missing releases due to upstream changes

### ISSUE THREAD

**Issue by pal1000** [avian2/noscript#13]
I found these releases:

NS10
- 10.1.7rc1;
- 10.1.7rc2;

NS5
- 5.1.8.5rc2;
- 5.1.8.5rc3.

**Comment by avian2** [comment]
Hm. This last commit also doesn't look right: https://github.com/avian2/noscript/commit/fa01ea95f206f73254e918dd2d4dcb41e1655e93

I guess there should be two branches, one for pre-webextension 5.1 series and another one for the new 10.1 series.

Thanks for reporting it. I've turned off automatic commits until I can look into it.

**Comment by avian2** [comment]
Not sure what is going on. At the moment, the beta channel on addons.mozilla.org only lists 10.1.6.6rc2 and 5.1.8.5rc1.

https://addons.mozilla.org/en-US/firefox/addon/noscript/versions/beta

**Comment by pal1000** [comment]
Yes, NS 5 and NS 10 were different branches since day 1. NS5 is XUL extension and NS10 is webextension.  Also [Mozilla addons beta channel has been deprecated](https://blog.mozilla.org/addons/2018/02/28/discontinuing-support-for-beta-versions/) so Giorgio Maone is self hosting the betas now. He enabled automatic updates in 5.1.8.5rc2 and 10.1.7rc2 respectively.

**Comment by polyzen** [comment]
Not only missing alpha/beta releases.

**Comment by polyzen** [comment]
https://noscript.net/feed covers at least stable releases.

**Comment by pal1000** [comment]
NoScript 5 both stable and RCs: [https://noscript.net/feed?classic](https://noscript.net/feed?classic).

**Comment by pal1000** [comment]
Until [https://noscript.net/feed?all](https://noscript.net/feed?all) gets fixed, you can find NS 10 betas here: [https://gist.github.com/chico-gordo/2452209fe8e8c573a3c68b8b8a36874d](https://gist.github.com/chico-gordo/2452209fe8e8c573a3c68b8b8a36874d).

**Comment by pal1000** [comment]
The feed that shows all releases including betas for both NoScript branches is [https://noscript.net/feed?t=a](https://noscript.net/feed?t=a). You can append `&c=<integer>` to specify how many releases including the latest to display. I think the default value for c is 100 if it's not specified.

**Comment by arthuredelstein** [comment]
@avian2, thank you for running this service. Would you be willing to post your noscript update script? I would like to patch it to follow https://noscript.net/feed?t=a if possible. I figure we can have a new branch for the 10.x series.

**Comment by avian2** [comment]
Everyone, thank you for reporting the problem and providing useful info on how to fetch new NoScript releases. Unfortunately due to lack of time I will no longer be able to maintain this repository further. I've updated the README to reflect that.

I've uploaded the scraping script that I used to update this repo here: https://github.com/avian2/noscript-scraper. @arthuredelstein if you'll setup another repository I would be happy to link to it from here.
